src/client.ts

```diff
@@ -167,6 +174,9 @@ export class SubscriptionClient {
 
     let opId: string;
 
+    this.activeSubscriptions += 1;
```

Comment:
This should be next to where `this.operations[opId] =` is done to ensure the reference counting it done correctly. Or possibly the check should just be against `Object.keys(this.operations).length` to avoid needing to do reference counting.

Reply:
Done, we are not counting `activeSubscriptions` anymore and relying on `Object.keys(this.operations).length` to test inactivity.